cli/graph_util.rs

```diff
@@ -165,6 +170,11 @@ impl GraphData {
     self.modules.iter()
   }
 
+  /// Gets if the graph had a "node:" specifier.
+  pub fn has_node_specifier(&self) -> bool {
```

Comment:
If we were to support `import fs from "fs"` without `node:` prefix this becomes too specific, maybe rename `has_node_builtin_specifier`?

Reply:
I think we'd probably map "fs" -> "node:fs" before it got here, but renaming sounds ok.